packages/@ember/test/index.js

```diff
@@ -1,11 +1,27 @@
-import { Test } from 'ember-testing';
+import require, { has } from 'require';
 
-const {
-  registerAsyncHelper,
-  registerHelper,
-  registerWaiter,
-  unregisterHelper,
-  unregisterWaiter,
-} = Test;
+export let registerAsyncHelper;
+export let registerHelper;
+export let registerWaiter;
+export let unregisterHelper;
+export let unregisterWaiter;
 
-export { registerAsyncHelper, registerHelper, registerWaiter, unregisterHelper, unregisterWaiter };
+if (has('ember-testing')) {
+  let { Test } = require('ember-testing');
+
+  registerAsyncHelper = Test.registerAsyncHelper;
+  registerHelper = Test.registerHelper;
+  registerWaiter = Test.registerWaiter;
+  unregisterHelper = Test.unregisterHelper;
+  unregisterWaiter = Test.unregisterWaiter;
+} else {
+  let testingNotAvailableMessage = () => {
+    throw new Error('Attempted to use test utilities, but `ember-testing` was not included');
```

Comment:
probably this should include a note that the user may want to wrap their usage in `if (DEBUG) {}` or in `if (Ember.testing) {}`, otherwise lgtm.